lib/ui/widgets/homeView/update_confirmation_sheet.dart

```diff
@@ -98,7 +100,7 @@ class UpdateConfirmationDialog extends StatelessWidget {
                     ),
                   ),
                   Padding(
-                    padding: const EdgeInsets.only(left: 24.0, bottom: 12.0),
+                    padding: const EdgeInsets.only(top: 12.0, left: 24.0, bottom: 12.0),
```

Comment:
@TheAabedKhan I am not sure if this is the relevant file but see https://github.com/ReVanced/revanced-manager/pull/1611#pullrequestreview-1857926227 for more information.

Reply:
> @TheAabedKhan I am not sure if this is the relevant file but see [#1611 (review)](https://github.com/ReVanced/revanced-manager/pull/1611#pullrequestreview-1857926227) for more information.

The padding? The sheet looks fine to me.

![image](https://github.com/ReVanced/revanced-manager/assets/93124920/fa71f791-2f18-42b6-a290-59f7c4a7de01)

Reply:
Done